libs/model/src/topic/GetTopics.query.ts

```diff
@@ -0,0 +1,40 @@
+import { type Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { FindAndCountOptions } from 'sequelize';
+import { models } from '../database';
+import { TopicAttributes } from '../models/index';
+import { removeUndefined } from '../utils';
+import { formatSequelizePagination } from '../utils/paginationUtils';
+
+export function GetTopics(): Query<typeof schemas.GetTopics> {
+  return {
+    ...schemas.GetTopics,
+    auth: [],
+    secure: false,
+    body: async ({ payload }) => {
+      const { community_id, topic_id, include_threads } = payload;
+
+      const includeArray = [];
+      if (include_threads) {
+        includeArray.push({
+          model: models.Thread,
+          as: 'threads',
+        });
+      }
+
+      const { count, rows: topics } = await models.Topic.findAndCountAll({
+        where: removeUndefined({ community_id, id: topic_id }),
+        include: includeArray,
+        ...formatSequelizePagination(payload),
+        paranoid: false,
+      } as unknown as FindAndCountOptions<TopicAttributes>);
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We basically have a structural issue here where the TopicAttributes contains the type for what sequelize needs from the developer.

But the zod types represent the type in the database.

This conflicts for example for Topic.description. In the database it is a non-null string that defaults to ''. The issue is that the TopicAttributes it is an optional string (becuase the developer does not need to supply it). But the actual return type is a non-optional string. I am unsure how to resolve this, and for that reason have typed it as any.

Reply:
This issue will happen with any attribute with a default value in the DB (optional when creating a record). We should establish a good pattern for handling these situations as we are bound to come across them again. @Rotorsoft @rbennettcw

Reply:
That said @kurtisassad can we not coerce the type to schemas.GetTopics.output instead of `any`?

Reply:
Yeah you are right, made the change